var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -5,7 +5,7 @@
 from spack import *
 
 
-class Kokkos(CMakePackage, CudaPackage):
+class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
```

Comment:
I think this is the first package I've seen with 3 base classes, exciting!

Reply:
Check out `raja` for another one :)